### ISSUE TITLE
How to get the information based on practice 

### ISSUE THREAD

**Issue by hardikplenar** [FokkeZB/libreview-unofficial#2]
Do you know how to get the connection based on the practice? Basically I have two practices but the patients are not connected to my account its connected to the practices so is it possible to fetch the log book of those patients?

**Comment by FokkeZB** [comment]
I have no idea. Did you try to login and use the connections endpoint to see what you get there?

**Comment by hardikplenar** [comment]
Yes the connections are empty so just wanted to know if we can have any filter in the connection or any parameter which we can pass in the connection endpoint which gives the connections based on practices?

**Comment by FokkeZB** [comment]
I have no idea. To figure that out, you would need to use the LibreView (web) app that your practice uses, and intercept network requests to hopefully find out.
